transformer_lens/loading_from_pretrained.py

```diff
@@ -155,6 +155,8 @@
     "meta-llama/Llama-3.2-3B",
     "meta-llama/Llama-3.2-1B-Instruct",
     "meta-llama/Llama-3.2-3B-Instruct",
+    "meta-llama/Llama-3.1-70B",
+    "meta-llama/Llama-3.1-8B",
     "Baidicoot/Othello-GPT-Transformer-Lens",
```

Comment:
We do also want to make sure we are including the Instruct version of the models. Those models are very popular in research. They should be relatively similar to add.